### PR TITLE
keepLogs configuration value

### PR DESCRIPTION
**Pull request by acollierr17**
This is in sync with the new `keepLogs` configuration that's been introduced over at https://github.com/suggestionsbot/suggestions-bot/pull/55

### CODE REVIEW

**Comment by acollierr17** [comment]
@Davi-the-Mudkip You have any grammar things you like me to add on/consider here?

File: docs/docs/managing-suggestions.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ If you haven't already done so, please refer to the [Getting Started](README.md#
 
 Suggestions are approved via the sID. Once you have it, run the command `approve <sID|message ID>`. Afterward, series of actions will happen:
 
-* The suggestion will update as "Approved" in the suggestions channel and shortly delete itself afterward.
+* The suggestion will update as "Approved" in the suggestions channel and its results will be logged.
 * The user will receive a DM that you approved their suggestion.
-* The results of that suggestion will be logged in the suggestions logs channel.
+* The results of that suggestion will either be posted to the suggestion logs channel or the suggestion message will be kept with its results displayed. (dependent on [`config keepLogs`](configuration.md#keep-logs))
```
Comment (Davi-the-Mudkip) [inline docs/docs/managing-suggestions.md:49]: For the sake of parallel structure:

```suggestion
* The results of that suggestion will either be posted to the suggestion logs channel or be kept in the suggestion message. (dependent on [`config keepLogs`](configuration.md#keep-logs))
```

File: docs/docs/managing-suggestions.md [hunk 2]
```diff
@@ -67,7 +67,12 @@ If responses are set to `true` via the usage of the [setresponses](commands.md#s
 
 ## Suggestions Logging
 
-When a suggestion is approved or rejected, its results are logged to the suggestions logs channel. If you have not set one up yet, refer to the [Getting Started](README.md#setting-up-the-bot) section. In the logs channel, you'll find this information for each result:
+When a suggestion is approved or rejected, one of two things will happen, depending on if you have a suggestions logs channel set and if [`config keepLogs`](configuration.md#keep-logs) is enabled (by default, it's `false`).
+
+- If you have no suggestion logs channel set or `config keepLogs` is true, then the suggestion message will be updated with its results and the reactions removed.
+- If you have a suggestion logs channel set and `config keepLogs` is false, The bot will delete the original suggestion message and its results posted to the set logs channel. You'll find the following information in the log message either way:
```
Comment (Davi-the-Mudkip) [inline docs/docs/managing-suggestions.md:73]: A small typo:
```suggestion
- If you have a suggestion logs channel set and `config keepLogs` is false, the bot will delete the original suggestion message and its results will be posted to the set logs channel. You'll find the following information in the log message either way:
```

File: docs/docs/managing-suggestions.md [hunk 3]
```diff
@@ -67,7 +67,12 @@ If responses are set to `true` via the usage of the [setresponses](commands.md#s
...
+
+If you wish to set up a suggestion logs channel, you may refer to the [Getting Started](README.md#setting-up-the-bot) section of the documentation.
```
Comment (Davi-the-Mudkip) [inline docs/docs/managing-suggestions.md]: This line should appear after the information list.

**Comment by Davi-the-Mudkip** [comment]
6

File: docs/docs/configuration.md [hunk 4]
```diff
@@ -118,3 +118,19 @@ When disabling DM responses, the bot won't DM the user on these actions:
...
+## Keep Logs
+Usage: `config keepLogs` or `config keepLogs [true|false]`
+
+You can either view if the suggestion message stays when approved/rejected or enable/disable the feature if you wish.
+
+When enabling this feature, a number of things will happen (in this order) after the suggestion is approved/rejected:
+
+1) The suggestion message will be updated with its status and result
+2) The reactions will be removed from the message
+3) The submitter (if DM responses are enabled) will be DMed that their suggestion has been processed
```
Comment (Davi-the-Mudkip) [inline docs/docs/configuration.md]: These list items should end with a period as they're complete sentences:
```suggestion
1) The suggestion message will be updated with its status and result.
2) The reactions will be removed from the message.
3) The submitter (if DM responses are enabled) will be DMed that their suggestion has been processed.
```

**Review by Davi-the-Mudkip — CHANGES_REQUESTED** [review]
Here are my comments:

File: docs/docs/managing-suggestions.md (hunk 3 above)
Comment (acollierr17) [reply docs/docs/managing-suggestions.md]: Should I put it after line 49?

Comment (Davi-the-Mudkip) [reply docs/docs/managing-suggestions.md]: No, you should put it after line 83.

Comment (acollierr17) [reply docs/docs/managing-suggestions.md]: Got it,

**Review by acollierr17 — COMMENTED** [review]
All changes from the code review have been resolved. Should be good to go.

**Pull request merged**